controller/script/simulate-proxy/main.go

```diff
@@ -119,8 +133,117 @@ var (
 	}
 )
 
-func randomPort() uint32 {
-	return ports[rand.Intn(len(ports))]
+// generateProxyTraffic randomly creates metrics under the guise of a single conduit proxy routing traffic.
+// metrics are generated for each proxyMetricCollector.
+func (s *simulatedProxy) generateProxyTraffic(podList []*v1.Pod) {
+	proxyPodName := strings.Split(s.pod.GetName(), "-")[0]
```

Comment:
is this supposed to be the deployment name?  I think you can get this from `s.pod.GetOwner.GetName()` or something like that